### PR TITLE
fix(editor): Disable all modifying keybindings when the canvas is in read-only mode (no-changelog)

### PR DESCRIPTION
**Pull request by alexgrozav**
## Summary

N/A

## Related Linear tickets, Github issues, and Community forum posts

https://linear.app/n8n/issue/N8N-7785/multiple-workflows-created-from-version-history

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/editor-ui/src/views/NodeView.v2.vue [hunk 1]
```diff
@@ -625,6 +625,8 @@ function onPinNodes(ids: string[], source: PinDataSource) {
 }
 
 async function onSaveWorkflow() {
+	if (isCanvasReadOnly.value) return;
```
Comment (netroy) [inline packages/editor-ui/src/views/NodeView.v2.vue:628]: could we instead not even emit the `save:workflow` event by adding a `props.readOnly` check [here](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/canvas/Canvas.vue#L153) instead?

Comment (alexgrozav) [reply packages/editor-ui/src/views/NodeView.v2.vue:628]: Great suggestion! I'm on it.

**Review by netroy — DISMISSED** [review]

**Review by netroy — DISMISSED** [review]

**Review by netroy — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.63.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.63.0)

**Comment by janober** [comment]
Got released with [`n8n@1.63.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.63.0)

**Pull request merged**